keyboards/yushakobo/primer16/keymaps/default/keymap.c

```diff
@@ -0,0 +1,61 @@
+/* Copyright 2022 yushakobo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT(
+  //,-----------------------------------|
+                 KC_F2,  KC_EQL,
+  //|--------+--------+--------+--------|
+        KC_P7,   KC_P8,   KC_P9, KC_PSLS,
+  //|--------+--------+--------+--------|
+        KC_P4,   KC_P5,   KC_P6, KC_PAST,
+  //|--------+--------+--------+--------|
+        KC_P1,   KC_P2,   KC_P3, KC_PMNS,
+  //|--------+--------+--------+--------|
+        KC_P0, KC_PDOT, KC_PENT, KC_PPLS
+  //|-----------------------------------|
+  )
+};
+
+//A description for expressing the layer position in LED mode.
```

Comment:
rgb lighting layers could be used here instead, as it wouldn't need the effect range set, or any of that, and could be moved into the keyboard's c file and overwritten by users.

https://docs.qmk.fm/#/feature_rgblight?id=lighting-layers

Reply:
Fixed as suggested